docs/2023/AI Tools/Deployment Optimization/updates/2023-08-04.md

```diff
@@ -4,13 +4,9 @@ author: Manas Sivakumar
 ---
 
 ## Milestones
-- [ ] Give the description about Milestone 1
-- [ ] Give the description about Milestone 2
-- [ ] Give the description about Milestone 3
-- [ ] Give the description about Milestone 4
-
-## Screenshots / Videos 
-
+- Fix generate.sh and ansible script issues
+  
 ## Contributions
-
```

Comment:
Please add your contribution in terms of PR.

Reply:
I don't have any prs for this week